website/client/src/components/groups/inviteModal.vue

```diff
@@ -152,12 +152,9 @@ export default {
   computed: {
     ...mapState({ user: 'user.data' }),
     cannotSubmit () {
-      const filteredInvites = filter(
-        this.invites,
-        invite => invite.text.length > 0 && !invite.valid,
-      );
-      if (filteredInvites.length > 0) return true;
-      return false;
+      const filledInvites = filter(this.invites, invite => invite.text.length);
+      const validInvites = filter(filledInvites, invite => invite.valid);
+      return !filledInvites.length || validInvites.length !== filledInvites.length;
```

Comment:
```suggestion
      const filledInvites = this.invites.filter(invite => invite.text.length);
      return !filledInvites.length || filledInvites.some(invite => !invite.valid);
```

I think for arrays it's better to prefer `Array.prototype.filter` instead of lodash's `filter` method, since it's closer to ES native and (IMO reads nicer). I also think using the `Array.prototype.some` method reads easier than the length check.

Reply:
Alternatively you could do `!filledInvites.length || !filledInvites.every(invite => invite.valid)`, but I think having the negation inside the arrow function reads easier.

Reply:
Agreed, where possible we should use ES native methods (these are all available on all modern browsers but even if they were not babel would take care of them and inject a polyfill)

Reply:
Yeah, I prefer using native array methods myself too, just didn't know what was the reason lodash methods were used there before so just kept the same style. But I can change them to native.